kernel_gateway/gatewayapp.py

```diff
@@ -198,12 +206,23 @@ def init_configurables(self):
         if self.seed_uri is not None:
             self.seed_notebook = self._load_notebook(self.seed_uri)
 
-        self.kernel_manager = SeedingMappingKernelManager(
-            parent=self,
-            log=self.log,
-            connection_dir=self.runtime_dir,
-            kernel_spec_manager=self.kernel_spec_manager
-        )
+        if self.api:
+            with open(self.api) as nb_fh:
+                notebook = nbformat.read(nb_fh, 4)
```

Comment:
`_load_notebook` always reads `self.seed_uri` instead of the `uri` that's passed in, which looks more like a bug than something intentional now. Will fix that and use it.